PWGHF/Core/DecayChannels.h

```diff
@@ -0,0 +1,183 @@
+enum DecayChannel : int8_t {
+  // D0
+  D0ToPiK = 1, // π+ K−
+  D0ToPiKPi0,  // π+ K− π0
+  D0ToPiPi,    // π+ π−
+  D0ToPiPiPi0, // π+ π− π0
+  D0ToKK,      // K+ K−
+  //
+  LastChannel
+};
+/// @brief 2-prong candidates: resonant channels
+enum DecayChannelResonant : int8_t {
+  // D0
+  D0ToRhoplusPi = 1, // ρ+ π−
+  D0ToRhoplusK,      // ρ+ K−
+  D0ToKstar0Pi0,     // anti-K*0 π0
+  D0ToKstarPi,       // K*− π+
+  //
+  LastChannelResonant
+};
+} // namespace hf_cand_2prong
+
+namespace hf_cand_3prong
+{
+/// @brief 3-prong candidates: non-resonant channels
+enum DecayChannel : int8_t {
+  // D+
+  DplusToPiKPi = 1, // π+ K− π+
+  DplusToPiKPiPi0,  // π+ K− π+ π0
+  DplusToPiPiPi,    // π+ π− π+
+  DplusToPiKK,      // π+ K− K+
+  // Ds+
+  DsToPiKK,      // π+ K− K+
+  DsToPiKKPi0,   // π+ K− K+ π0
+  DsToPiPiK,     // π+ π− K+
+  DsToPiPiPi,    // π+ π− π+
+  DsToPiPiPiPi0, // π+ π− π+ π0
+  // D*+
+  DstarToPiKPi, // π+ K− π+ (from [(D0 → π+ K−) π+])
+  // Λc+
+  LcToPKPi,    // p K− π+
+  LcToPKPiPi0, // p K− π+ π0
+  LcToPPiPi,   // p π− π+
+  LcToPKK,     // p K− K+
+  // Ξc+
+  XicToPKPi,  // p K− π+
+  XicToPKK,   // p K− K+
+  XicToSPiPi, // Σ+ π− π+
+  //
+  LastChannel
+};
+/// @brief 3-prong candidates: resonant channels
+enum DecayChannelResonant : int8_t {
+  // D+
+  DplusToPhiPi = 1, // φ π+
+  DplusToKstar0K,   // anti-K*0 K+
+  DplusToAKstar0K,  // anti-K*0(1430) K+ // FIXME: What to call this?
+  DplusToRho0Pi,    // ρ0 π+
+  DplusToF2Pi,      // f2(1270) π+
+  // Ds+
+  DsToPhiPi,      // φ π+
+  DsToPhiRhoplus, // φ ρ+
+  DsToKstar0K,    // anti-K*0 K+
+  DsToKstar0Pi,   // anti-K*0 π+
+  DsToRho0Pi,     // ρ0 π+
+  DsToRho0K,      // ρ0 K+
+  DsToF2Pi,       // f2(1270) π+
+  DsToF0K,        // f0(1370) K+
+  DsToEtaPi,      // η π+
+  // Λc+
+  LcToPKstar0,        // p K*0(892)
+  LcToDeltaplusplusK, // Δ++ K−
+  LcToLPi,            // Λ(1520) π+
+  LcToLK,             // Λ(1520) K+
+  // Ξc+
+  XicToPKstar0, // p anti-K*0(892)
+  XicToPPhi,    // p φ
+  //
+  LastChannelResonant
+};
+} // namespace hf_cand_3prong
+
+namespace hf_cand_dstar
+{
+/// @brief D*+ candidates: non-resonant channels
+enum DecayChannel : int8_t {
+  // D*+
+  DstarToPiKPi = 1, // π+ K− π+ (from [(D0 → π+ K−) π+])
+  DstarToPiKPiPi0,  // π+ K− π+ π0 (from [(D0 → π+ K− π0) π+] or [(D+ → π+ K− π+) π0])
+  //
+  LastChannel
+};
+} // namespace hf_cand_dstar
+
+namespace hf_cand_beauty_neutral
+{
+/// @brief neutral-beauty candidates: non-resonant channels
+enum DecayChannel : int8_t {
+  // B0
+  B0ToDminusPi = 1, // D− π+
+  B0ToDminusK,      // D− K+
+  B0ToD0PiPi,       // anti-D0 π+ π−
+  // Bs0
+  BsToDsPi, // Ds− π+
+  BsToDsK,  // Ds− K+
+  // Λb0
+  LbToLcPi, // Λc+ π−
+  LbToLcK,  // Λc+ K−
+  LbToScPi, // Σc+ π−
+  LbToScK,  // Σc+ K−
+  //
+  LastChannel
+};
+/// @brief neutral-beauty candidates: resonant channels
+enum DecayChannelResonant : int8_t {
+  // B0
+  B0ToDminusRhoplus = 1, // D− ρ+
+  B0ToDstarminusPi,      // D*− π+
```

Comment:
These might not correspond to any channel in the "main ones" (for instance `B0ToDstarminusPi` can be a resonant channel for `B0ToD0barPiPi` but also a partly reconstructed decay for `B0ToDminusPi` if the Dstar- decays to D- Pi0 or Gamma).
To be consistent with the charm we should probably add the channels with Pi0 and Gamma in the `DecayChannel` enum what do you think?